src/design-tokens/mixins.css

```diff
@@ -116,29 +116,21 @@
   /**
    * Input error state
    */
-  &.eds-is-error {
+  &.error {
     border-color: var(--eds-theme-color-border-utility-error-strong);
   }
 
   /**
    * Disabled state
    */
-  &.eds-is-disabled {
+  &:disabled {
     background-color: var(--eds-theme-color-background-disabled);
     border-color: var(--eds-theme-color-border-disabled);
     color: var(--eds-theme-color-text-disabled);
     cursor: not-allowed;
   }
 
-  &::-webkit-input-placeholder {
-    @mixin placeholderStyles;
-  }
-
-  &::-moz-placeholder {
-    @mixin placeholderStyles;
-  }
-
-  &:-ms-input-placeholder {
```

Comment:
Oh yeah I saw these the other day when I was doing the textarea thingy.. Could you test this in Firefox just to make sure `::placeholder` is enough?

Reply:
Yup works in firefox, and also tested in safari, and confirmed by [caniuse](https://caniuse.com/css-placeholder)

![image](https://user-images.githubusercontent.com/86632227/186951417-9ba591af-eda7-46c1-b8bd-515b496469d2.png)

Reply:
Awesome, thanks for testing!